src/app/App.js

```diff
@@ -1,10 +1,38 @@
 // @flow
 
 import React from "react";
+import {BrowserRouter as Router, Route, NavLink} from "react-router-dom";
```

Comment:
This is fine with me for now. I suspect that eventually, instead of this import, we will want to DI `<App router={BrowserRouter} />` so that we can use a `MemoryRouter` (or similar) for testing.